core/templates/services/local-storage.service.spec.ts

```diff
@@ -126,6 +126,20 @@ describe('LocalStorageService', () => {
         .toBeNull();
     });
 
+    it('should correctly save and retrieve sign up section preference', () => {
+      localStorageService.updateEndChapterSignUpSectionHiddenPreference('true');
+      expect(localStorageService.getEndChapterSignUpSectionHiddenPreference())
```

Comment:
Is it possible to have a pre-check for this?

Reply:
Done.